sdk/src/utils.ts

```diff
@@ -308,3 +309,20 @@ export const sqrtPriceToPrice = (sqrtPrice: SqrtPrice): Price => {
 export const priceToSqrtPrice = (price: Price): SqrtPrice => {
   return sqrt(price * getSqrtPriceDenominator())
 }
+
+interface LiquidityParsedTick {
+  liquidity: Liquidity
+  index: bigint
+}
+
+export const parseLiquidityOnTicks = (ticks: Tick[]): LiquidityParsedTick[] => {
```

Comment:
according to name in struct name you can rename this function: `calculateLiquidityBreakpoints`